src/resources/js/tickets-block.js

```diff
@@ -1103,116 +1065,116 @@ tribe.tickets.block  = {
 	 *
 	 * @since 4.11.0
 	 *
-	 * @return obj Tickets data object.
+	 * @returns {object} - Tickets data object.
 	 */
 	obj.getTicketsForCart = function() {
-		var tickets   = [];
-		var $cartForm = $( obj.modalSelector.cartForm );
+		const tickets = [];
+		let $cartForm = $( obj.modalSelector.cartForm );
 
 		// Handle non-modal instances
 		if ( ! $cartForm.length ) {
 			$cartForm = $( obj.selector.container );
 		}
 
-		var $ticketRows = $cartForm.find( obj.selector.item );
+		const $ticketRows = $cartForm.find( obj.selector.item );
 
 		$ticketRows.each(
 			function() {
-				var $row        = $( this );
-				var ticket_id    = $row.data( 'ticketId' );
-				var qty          = $row.find( obj.selector.itemQuantityInput ).val();
-				var $optoutInput = $row.find( '[name="attendee[optout]"]' );
-				var optout       = $optoutInput.val();
+				const $row = $( this );
+				const ticketId = $row.data( 'ticketId' );
+				const qty = $row.find( obj.selector.itemQuantityInput ).val();
+				const $optoutInput = $row.find( '[ name="attendee[ optout ]" ]' );
```

Comment:
Is this find() intentional? Seems like an unintentional side effect here.

Reply:
Yes. Intentional as it gets us the input specific to that **row**.

Reply:
Note the only changes on the above lines is var -> const, camelcase, and some whitespace.

Reply:
But thanks for the comment - because it reminded me that there were some places I could simplify the code by using template literals!

Reply:
I was referring to here and other places about the white space in the selectors themselves. `input[ type="something" ]` where we changed those selectors in the syntax

Reply:
So the spaces in the selector syntax work for jQuery? That’s what I wanted to be sure of here. I wasn’t sure if the extra whitespace mattered for that syntax.

Reply:
Oh yes - for reference (since we discussed elsewhere as well) they do in fact work.